### ISSUE TITLE
Project visualization X-Axis has no parameters

### ISSUE THREAD

**Issue by maifeng** [comet-ml/issue-tracking#274]
Hello, I ran a simple hyperparameter tuning experiment by adapting the sample Keras code (the old Optimizer API in the sample code is deprecated). All went well, except that I cannot see any hyperparameter in the X-Axis in project visualization. I'd like to plot a graph of Dropout rate v.s. validation score. 

The hyperparameters are recorded (I can view them in the individual experiment, see Fig 2), but all I see on the project chart dropdown menu are items from **TIME-SERIES** and **METRICS** (Fig 1).  Am I missing something in my code? Thanks. 

**Fig 1. Cannot see Hyperparameter in Chart**
![image](https://user-images.githubusercontent.com/1271491/60750043-ed6ce100-9f70-11e9-8507-346fb8fcab26.png)

**Fig 2. Hyperparameters are recorded**
![image](https://user-images.githubusercontent.com/1271491/60750051-01184780-9f71-11e9-9d4d-774ce2028bd3.png)

**Code**
```
import sys
from comet_ml import (
    Experiment,
    Optimizer
)

import keras
from keras.datasets import mnist
from keras.models import Sequential
from keras.layers import Dense, Dropout
from keras.optimizers import RMSprop

def main():

    num_classes = 10
    # the data, shuffled and split between train and test sets
    (x_train, y_train), (x_test, y_test) = mnist.load_data()
    x_train = x_train.reshape(60000, 784)
    x_test = x_test.reshape(10000, 784)
    x_train = x_train.astype("float32")
    x_test = x_test.astype("float32")
    x_train /= 255
    x_test /= 255
    print(x_train.shape[0], "train samples")
    print(x_test.shape[0], "test samples")
    # convert class vectors to binary class matrices
    y_train = keras.utils.to_categorical(y_train, num_classes)
    y_test = keras.utils.to_categorical(y_test, num_classes)

    config = {
        "algorithm": "bayes",
        "parameters": {
            "first_layer_units": {"type": "discrete", "values": [2, 4, 8]},
            "second_layer_units": {"type": "integer", "min": 1, "max": 10},
            "dropout": {"type": "double", "scalingType": "linear", "min": 0.3, "max": 0.9}
        },
        "spec": {
            "metric": "score",
            "objective": "maximize",
            "maxCombo": 10,
        },
        "name": "My Bayesian Search"
        }

    # Gets API from config or environment:
    opt = Optimizer(config, project_name="test2")
    for experiment in opt.get_experiments():
        score = train(x_train, y_train, x_test, y_test, 2, 480, experiment)
        experiment.log_metric("score", score)

def build_model_graph(experiment):
    model = Sequential()
    model.add(Dense(experiment.get_parameter("first_layer_units"), activation="sigmoid", input_shape=(784,)))
    model.add(Dense(experiment.get_parameter("second_layer_units"), activation="sigmoid"))
    model.add(Dropout(experiment.get_parameter("dropout")))
    model.add(Dense(10, activation="softmax"))
    model.compile(
        loss="categorical_crossentropy", optimizer=RMSprop(), metrics=["accuracy"]
    )
    return model


def train(x_train, y_train, x_test, y_test, epoch, batch_size, experiment):
    # Define model
    model = build_model_graph(experiment)
    model.fit(
        x_train,
        y_train,
        batch_size=batch_size,
        epochs=epoch,
        validation_data=(x_test, y_test),
    )
    score = model.evaluate(x_test, y_test, verbose=0)[1]
    return score


if __name__ == "__main__":
    main()

```

**Comment by gidim** [comment]
Hey @maifeng  - thanks for reporting!
You can get a much more robust view with the parallel coordinates chart. See example here:
https://www.comet.ml/cometpublic/parameter-space-exploration

We're also adding support for scatter plots as your requested.

**Comment by gidim** [comment]
@maifeng  - you can now plot this using the "Scatter plot" option
